Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ClipboardHelper/ClipboardHelperCode.bind

```diff
@@ -0,0 +1,25 @@
+// Set text into clipboard.
+ClipboardHelper.SetText("Hello World");
+
+// Get text from clipboard.
+var text = ClipboardHelper.GetTextAsync();
```

Comment:
Is is not `var text = await ClipboardHelper.GetTextAsync(); ` ? Using `await` keyword.